ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/validator/preexecution/PreExecutingSubmissionValidatorSpec.scala

```diff
@@ -174,9 +209,11 @@ object PreExecutingSubmissionValidatorSpec {
   private final case class TestValue(value: Option[DamlStateValue])
 
   private object TestValue {
+
     implicit object `TestValue has DamlStateValue` extends HasDamlStateValue[TestValue] {
       override def damlStateValue(value: TestValue): Option[DamlStateValue] = value.value
     }
+
```

Comment:
```suggestion
```

Reply:
scalafmt did that on save, doesn't it for you?

Reply:
Done.